data.json

```diff
@@ -1378,5 +1378,11 @@
     "githubUsername": "Vanshikapandey30",
     "imageUrl": "https://user-images.githubusercontent.com/70807500/135111984-f6c4f540-949b-451c-8fb3-22d5daa07575.png",
     "issueNumber": 738
+  },
+    {
```

Comment:
Please can you tab this back a space, so it is in line with the other brackets?

Reply:
done

Reply:
Thank you  @DarshGupta1910